### PR TITLE
https://github.com/classy-org/classy-node/issues/102

### PR DESCRIPTION
**Pull request by brownjava**
Make _createFullPath OS-agnostic

### CODE REVIEW

File: test/test_createFullPath.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    ['/a///b', true, '/c/d/', '/a/b'],
+    ['/a//b', true, '//c/d/', '/a/b'],
+    ['/a///b', true, 'c/d/', '/a/b'],
+    ['a/b///', true, 'c/d/', 'a/b/'],
+  ];
+
+  const runTest = (resolvedPath, isAuthRequest, basePath, expectedOutput) => {
+    const testName = `_createFullPath(${resolvedPath}, ${isAuthRequest}, ${basePath})`
+      + ` => ${expectedOutput}`;
+    it(testName, async () => {
```
Comment (mlingner) [inline test/test_createFullPath.js]: I think this needs to be a regular function.

**Comment by brownjava** [comment]
What version of node is expected by this project?  There's no .nvmrc.

**Comment by mlingner** [comment]
Unfortunately it's 4 right now according to travis.yml. We should follow up with a major version release which upgrades our Node version and transpiles async/await code with a more refined Babel config.

**Comment by mlingner** [comment]
This is now merged and published under v2.6.3.

**Pull request merged**